tokio/src/stream/any.rs

```diff
@@ -0,0 +1,45 @@
+use crate::stream::Stream;
+
+use core::future::Future;
+use core::pin::Pin;
+use core::task::{Context, Poll};
+
+/// Future for the [`any`](super::StreamExt::any) method.
+#[derive(Debug)]
+#[must_use = "futures do nothing unless you `.await` or poll them"]
+pub struct AnyFuture<'a, St: ?Sized, F> {
```

Comment:
Super minor nit. Should this be named `Any`? None of the other futures have the `Future` suffix.

Reply:
I'm afraid that this will cause a collision with std's `Any` should we make combinator structs public. Also this is a follow up to the discord discussion around the benefits of encoding in the name the fact that this combinator represents a future.

Reply:
**most** people won't import this. The naming collision doesn't seem like a huge danger at this point. Either way, not a huge deal.